capstone/src/main/webapp/statistics_script.js

```diff
@@ -66,7 +110,7 @@ function loadBubbleChart(listA, listB, listC, listD) {
   const stats = new google.visualization.DataTable();
 
   stats.addColumn('string', 'ID');
-  stats.addColumn('number', 'Days');
+  stats.addColumn('number', 'Day in July');
```

Comment:
More of a data visualization stylistic issues but have you considered having the axis be time instead?

Reply:
I wanted to make it more specific so that it would be easier to comprehend